packages/ui/src/components/cell/cell.native.tsx

```diff
@@ -1,13 +1,6 @@
 import { ResponsiveValue, useTheme } from '@shopify/restyle';
 
-import {
-  Box,
-  ChevronRightIcon,
-  IconProps,
-  Text,
-  Theme,
-  TouchableOpacity,
-} from '@leather.io/ui/native';
+import { Box, ChevronRightIcon, IconProps, Text, Theme, TouchableOpacity } from '../../../native';
```

Comment:
This is really just moving the existing `Cell` we had out. 

I can do more work to refactor it and get it to receive `ItemLayout` instead

Reply:
No rush here, I can work with the existing `Cell`, if some of this code is just a dup of `ItemLayout` then let's get that change in here. The stuff I'm doing is just setting up the UI, so not difficult to swap out. I think it's better to have these components setup correctly.

Reply:
I'll merge this to move `Cell` out of the app code but will then refactor it more if it's the same as item layout and add this variant - https://github.com/leather-io/mono/pull/398#pullrequestreview-2268577677